tests/fsharp/test-framework.fs

```diff
@@ -164,7 +164,7 @@ let config configurationName envVars =
 #if NET472
     let fscArchitecture = "net472"
     let fsiArchitecture = "net472"
-    let fsharpCoreArchitecture = "net45"
+    let fsharpCoreArchitecture = "netstandard2.0"
```

Comment:
This line can be pulled out of the `#if NET472` block since it's the same for both scenarios.